src/fonduer/candidates/candidates.py

```diff
@@ -283,6 +280,17 @@ def apply(  # type: ignore
                     for mention in candidate_class.mentions
                 ]
             )
+            # Construct a set of stable ids of candidates.
+            set_of_stable_ids = set()
+            if hasattr(doc, candidate_class.__tablename__ + "s"):
```

Comment:
Can we have a documentation here to mention why `+ "s"`?

Reply:
I added a comment to explain the reason.
See https://github.com/HazyResearch/fonduer/pull/368/files#diff-dd77b82c8178c68913da115c82628c3dR56.